apps/builder/app/builder/features/sidebar-left/panels/pages/pages.tsx

```diff
@@ -17,8 +17,8 @@ import {
 import {
   ChevronRightIcon,
   MenuIcon,
-  NewPageIcon,
```

Comment:
Delete the old icon?

Reply:
still in the collection, bight be useful somewhere else